build.gradle

```diff
@@ -339,6 +339,10 @@ List<String> jacocoExclusions = [
         'org.opensearch.ad.task.ADTaskManager',
         'org.opensearch.ad.task.ADTaskCacheManager',
         'org.opensearch.ad.transport.ForwardADTaskTransportAction',
+        'org.opensearch.ad.task.ADHCBatchTaskCache',
+        'org.opensearch.ad.task.ADBatchTaskRunner',
+        'org.opensearch.ad.transport.ForwardADTaskRequest',
+        'org.opensearch.ad.task.ADBatchTaskCache'
```

Comment:
Any specific reasons to exclude those classes?

Reply:
This is to make sure the build can pass locally, will add more UT to increase the coverage.